classes/patterns/class-action.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+
+		return [
+			'title'      => __( 'Action', 'planet4-blocks-backend' ),
+			'categories' => [ 'layouts' ],
+			'content'    => '
+				<!-- wp:group {"className":"block"} -->
+					<div class="wp-block-group">
+						<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image"} -->
+							<div class="wp-block-media-text is-stacked-on-mobile">
+								<figure class="wp-block-media-text__media">
+									<img src="' . $media_link . '" alt="' . __( 'Default image', 'planet4-blocks-backend' ) . '"/>
+								</figure>
+								<div class="wp-block-media-text__content">
+									<!-- wp:gravityforms/form /-->
+								</div>
+							</div>
+						<!-- /wp:media-text -->
+						<!-- wp:group {"backgroundColor":"grey-05","align":"full","className":"is-style-large-padding"} -->
+							<div class="wp-block-group alignfull has-grey-05-background-color has-background is-style-large-padding">
+								<!-- wp:group {"className":"container"} -->
+									<div class="wp-block-group container">
+										' . SideImageWithTextAndCta::get_config(
+										[
+											'media_position'    => 'right',
```

Comment:
I just realized there's a mistake in the `SideImageWithTextAndCta`, the media position class is used [instead of the mediaPosition attribute](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/patterns/class-sideimagewithtextandcta.php#L45), which confuses the media-text UI. You can add a tiny fix in this PR or we can open a different one

Reply:
Another good catch! [Pushed a fix](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/b0688384075ca122da35a22b50ff8936e3952239)